data/swtpm/ssh_port_chk_script

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+ip_addr=""
```

Comment:
A good practice in general for bash scripts is to call `set -euo pipefail` to exit on unexpected errors and mistakes

Reply:
nice comment, let me fix it

Reply:
Added